### ISSUE TITLE
Accumulo 1.9.x does not work with Hadoop 3.x

### ISSUE THREAD

**Issue by mikewalch** [apache/accumulo#569]
I tried running Accumulo 1.9.2 with Hadoop 3.1.0 and it failed due to Hadoop 3.1.0 not including commons-configuration 1.6 (Hadoop has moved to commons-configuration2).  Accumulo 2.0.0-SNAPSHOT fixes this by including commons-configuration in its tarball.  This is something Accumulo 1.9.x could do also if there is a desire for Accumulo 1.9.x to work with Hadoop 3.

**Comment by ctubbsii** [comment]
If you build the Accumulo tarball with the Hadoop 3 profile, `-Dhadoop.profile=3`, it will include commons-configuration 1.6. There was a JIRA issue, https://issues.apache.org/jira/browse/ACCUMULO-4830 , which proposed creating a single binary tarball compatible with both... which might be easy if we bundle everything we need, and have different config templates for Hadoop 2 and 3, which list their class path items directly rather than use wildcards.

When I tried using 1.9.x with Hadoop 3.0.2, it worked as long as I built with the Hadoop 3 profile, but I couldn't get it to work with 3.1.0, and I wasn't able to figure out why.
